.github/workflows/publish.yaml

```
@@ -8,19 +8,28 @@ jobs:
   build:
     name: Build distribution package
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v4
         with:
           persist-credentials: false
+      - name: Fetch tags
```

This can also be skipped for release requests:
```suggestion
      - name: Fetch tags
        if: github.ref_type != 'tag'
```